### DISCUSSION TITLE
Notification icon with number

### DISCUSSION THREAD

**Discussion by newtec-foggia** [laravel/nova-issues#4292] [Ideas & Feature Requests]
it would be interesting to see the number of unread notifications ..

![download](https://user-images.githubusercontent.com/83398776/169593683-091a36a5-7f0d-441d-9554-5bbcbdd344d6.jpg)

**Comment by szymon-szychulski** [comment]
How is the user supposed to know that there are new notifications if nothing happens because the icon changes to unread only with one notification and there is no unread notification counter to know how many there are...

**Comment by tmartty** [comment]
Any news on this? As mentioned, a user that has some pending/unread notifications currently has no way to know when a new notification arrives on the notification center. It would be useful to have the number of unread notifications within the red notification icon.
